main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"net"
+	"os"
+
+	pb "github.com/kubearmor/KVMService/protobuf"
+	"google.golang.org/grpc"
+)
+
+type server struct {
+	pb.UnimplementedKVMServer
+}
+
+func (s *server) RegisterAgentIdentity(ctx context.Context, in *pb.AgentIdentity) (*pb.Status, error) {
+	log.Println("Agent Identity is ", in.Identity)
+	os.Setenv("AGENT_IDENTITY", in.Identity)
```

Comment:
We should not be setting the env on KVMService. This needs to be removed.
We will have multiple agents talking to the same KVMService.

Reply:
Yes Rahul. currently we have not decided how the identity is stored. This value will be later moved to some DB/variable.